client/core/types.go

```diff
@@ -704,7 +710,10 @@ type dexAccount struct {
 	bonds         []*db.Bond // confirmed, and not yet expired
 	expiredBonds  []*db.Bond // expired and needing refund
 	tier          int64      // check instead of isSuspended
```

Comment:
`tier` is the last known tier that the server/network says we are.  This was established previously, but it will become much more clearer in reserves work I'm prepping to make public.

There's a concept of `bondedTier`, which would be what you are describing - a function of known bonds and their statuses.  Then their is effective tier, which is what server says we are.  Both of these concepts come into play, but this value is the deciding factor as it relates to how we expect server to interact with us.

On a related note, we have future work to do with including more penalty info, at least in the form of a single "tier adjustment" integer for this, but ideally with more description of the violations as server sees them.